js/app/libs.js

```diff
@@ -137,18 +214,18 @@ Fliplet.Registry.set('notification-inbox:1.0:app:core', function(data) {
     ts = ts || Date.now();
     opt = opt || {};
 
-    return getNewNotifications(ts)
+    return getLatestNotificationCounts(ts, opt)
```

Comment:
where is `opt` used?

Reply:
`opt` is passed to `getLatestNofificationCounts()` for which the options are used [here](https://github.com/Fliplet/fliplet-widget-notification-inbox/blob/47d714d087c53ba2709640df8dfebb78dea9282a/js/app/libs.js#L191-L191)